tool.go

```diff
@@ -498,7 +498,21 @@ func main() {
 	cmdServe.Run = func(cmd *cobra.Command, args []string) {
 		options.BuildTags = strings.Fields(*tags)
 		dirs := append(filepath.SplitList(build.Default.GOPATH), build.Default.GOROOT)
-		sourceFiles := http.FileServer(serveCommandFileSystem{options: options, dirs: dirs, sourceMaps: make(map[string][]byte)})
+		root := ""
```

Comment:
Since i'm new to Go, I'll take your word for it, but that is surprising to me. Is the explicitly typed form considered idiomatic even in the case of simple values? I recognize the value of leaning on zero values (a technique that I'm a fan of from C/C++), but in this case, the explicit assignment is shorter and no less clear. Am I missing something?